src/css/cs/items/_kitchensink.scss

```diff
@@ -376,6 +376,19 @@
     }
   }
 
+  // Style the text that appears in the HTMLArea on UC_STATUS_MSG_SBF
+  .uc-status-message {
+    border: 0;
+    box-shadow: none;
+    font-size: 16px;
+    font-weight: bold;
+    margin: 0;
+    outline: none;
```

Comment:
the page behaves the same to the non-screen-reading user as before...at least in my experience